woocommerce/woocommerce-gutenberg-products-block
# Add wide and full alignment support for legacy template block

This PR adds support for the wide and full alignment for the legacy template block.


Fixes #5229 

### Screenshots


![image](https://user-images.githubusercontent.com/4463174/146966725-51064733-5b68-42d5-b1d7-36e7972f33bc.png)
*editor side
![image](https://user-images.githubusercontent.com/4463174/146969960-a8a21ee5-e52b-478a-89fb-2e7eda7d7165.png)
*frontend side


### Testing


### Manual Testing

How to test the changes in this Pull Request:

Check out this branch

1. Open FSE editor.
2. Edit a legacy template (for example `Product Category Page`).
3. Check if you can change the alignment. 
4. Change it.
5. Check if the changes are also reflected on the frontend side.

You should test at least one between `Product Category Page`, `Product Archive Page` and `Product Tag Page`. 
You should test `Single Product Page`.



### Changelog

> Add support for the wide and full alignment for the legacy template block.
